base/strings/unicode.jl

```diff
@@ -174,14 +174,19 @@ function utf8proc_map(str::Union{String,SubString{String}}, options::Integer, ch
     return String(resize!(buffer, nbytes))
 end
 
-# from julia_charmap.h, used by julia_chartransform in the Unicode stdlib
+# Array of {original codepoint, replacement codepoint} normalizations
+# to perform on Julia identifiers, to canonicalize characters that
+# are both easily confused and easily inputted by accident.
+#
+# Important: when this table is updated, also update the corresponding table
+#            in src/flisp/julia_charmap.h
```

Comment:
I guess it's nicer to make it a doc string, using Markdown:

```suggestion
"""
`Dict` of `original codepoint => replacement codepoint` normalizations
to perform on Julia identifiers, to canonicalize characters that
are both easily confused and easily inputted by accident.

!!! warning
    When this table is updated, also update the corresponding table in `src/flisp/julia_charmap.h`.
"""
```